tests/ttnn/unit_tests/operations/test_unary.py

```diff
@@ -29,15 +31,16 @@ def run_unary_test(device, h, w, ttnn_function, torch_function, pcc=0.9999):
 
 def run_identity_test(device, h, w, data_type, pcc=0.9999):
     torch.manual_seed(0)
-
+    ttnn_function = ttnn.identity
```

Comment:
What is the purpose of this variable?

Reply:
It is used to get the golden function for identity op.